pkg/core/chain/synchronizer_test.go

```diff
@@ -78,6 +79,23 @@ func TestSyncProgress(t *testing.T) {
 	assert.Equal(resp.Progress, float32(50.0))
 }
 
+func TestSyncConcurrency(t *testing.T) {
+
```

Comment:
Style nit: please dont leave empty lines between function declarations and the function body - this helps keep the code style consistent.